contracts/TNT.sol

```diff
@@ -56,28 +82,44 @@ contract TNTContract is ERC721Enumerable, AccessControl {
      */
     function burnToken(uint256 tokenId) public {
         require(ownerOf(tokenId) == msg.sender, "You do not own this token");
-
         _burn(tokenId);
     }
 
     /**
-     * @dev Grants minter role to an account.
+     * @dev Overrides transfer functions to disable token transfers.
```

Comment:
Here you are forgetting to override the transfer function as well.

But, instead of overriding every function that causes a token transfer, it is better to follow the approach described here (https://medium.com/@0xcueto/how-to-create-a-non-transferable-nft-or-soulbound-nft-12a2a27cfac7).